pkg/config/config_test.go

```diff
@@ -684,6 +684,14 @@ process_config:
 	}
 }
 
+func TestComputeAPIDomain(t *testing.T) {
+	testDomain := "https://app.datadoghq.com"
+	matchDomain := "https://api.datadoghq.com"
+	apiDomain := ComputeAPIDomain(testDomain)
+
+	assert.Equal(t, apiDomain, matchDomain)
```

Comment:
nit: technically, it's `assert.Equal(t, expected, actual)`, so let's do:

```suggestion
	assert.Equal(t, matchDomain, apiDomain)
```

instead